.github/get-latest-python-version.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+import json
+import re
+import urllib.request as request
+
+
+def get_superior_version_tag(tag1: str, tag2: str) -> str:
+    tag1_split = tag1.split(".")
+    tag2_split = tag2.split(".")
+    for i in range(len(tag1_split)):
+        if int(tag1_split[i]) > int(tag2_split[i]):
+            return tag1
+        elif int(tag1_split[i]) < int(tag2_split[i]):
+            return tag2
```

Comment:
Might be safer to use https://packaging.pypa.io/en/latest/version.html#packaging.version.parse ?